test/installer/scripts/test_omsadmin.sh

```diff
@@ -1,42 +0,0 @@
-BASE_DIR=$1
-RUBY_TESTVERS=$2
```

Comment:
This test was for testing omsadmin.sh shell script with onboarding parameters. why you think it has to be deleted?